Source/Lib/DPX/DPX.cpp

```diff
@@ -213,11 +213,30 @@ bool dpx::Parse()
         return Error("Style (Descriptor / BitDepth / Packing / Endianess combination)");
     Style = DPX_Tested[Tested].Style;
 
+    // Slices count
+    // Computing optimal count of slices. TODO: agree with everyone about the goal and/or permit multiple formulas
+    // Current idea:
+    // - have some SIMD compatible slice count (e.g. AVX-512 has 16 32-bit blocks, let's take multiples of 16)
+    // - each slice has around 256 KiB of data, there is a similar risk of losing 1 LTO block (correction code per block, to be confirmed but looks like a 256 KiB block size is classic)

```

Comment:
If I remember carefully, LTFS 2.4 defines a block size of 524288 bytes.

Reply:
From my reading of [LTFS 2.4](https://www.snia.org/sites/default/files/technical_work/LTFS/LTFS_Format_2.4.0_TechPosition.pdf), I understand that "\<blocksize\>524288\</blocksize\>" is only an example.
I added a comment referencing it.
I think it does not change the suggestion of default slice count, for the moment it seems something average of all what people have in mind about slices (some people like less slices, @dericed prefers more slices i.e. IIRC he suggests 24 slices for SD 10-bit)

Reply:
I have in mind 512 KiB and, I guess, that’s what we are testing internally.

I’m not against many slices! I just suggest the formula in order to optimise the use of the available computing power.

Reply:
> I just suggest the formula in order to optimise the use of the available computing power.

The plan is (short term) to add option for setting your own count of slices, then (medium term) add a bench mode automating the search of the best config including slice count for a specific CPU.

But I am personally not for trying to find the best settings for a specific CPU, I am more in favor on having a general recommendation for being not too slow, having average compression, lot of slices, expecting that future CPU/GPU will be massively parallelized, as playback may be on other machine than the one used for compression. Anyway, anyone will have the possibility to tweak the FFV1 settings.

Reply:
P.S. – Don’t forget that when you do not impose the order, then the data will not be recorded in ascending order onto the LTO cartridge. To impose an order takes a little longer, but it may save a lot of spinning time when retrieving the information.

Reply:
>The plan is (short term) to add option for setting your own count of slices

+1